packages/vite/src/node/plugins/css.ts

```diff
@@ -521,21 +533,21 @@ async function compileCSS(
   const deps = new Set<string>()
 
   // 2. pre-processors: sass etc.
-  if (lang && lang in preProcessors) {
-    const preProcessor = preProcessors[lang as PreprocessLang]
+  if (lang && isPreProcessor(lang)) {
```

Comment:
**suggestion (non-blocking):** You could move the `falsy` check also into the `isPreProcessor` guard

Reply:
Good suggestion.